__tests__/api.test.js

```diff
@@ -48,6 +48,40 @@ describe("/api/topics", () => {
 	});
 });
 
+describe("/api/articles", () => {
+	test("GET STATUS 200: should return an array of articles objects with their respective properties.", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then((articles) => {
+				expect(articles.body.articles.length).toBe(13);
+				expect(articles.body.articles[0]).toMatchObject({
+					article_id: 3,
+					author: "icellusedkars",
+					title: "Eight pug gifs that remind me of mitch",
+					topic: "mitch",
+					created_at: convertTimestampToDate(1604394720000),
+					votes: 0,
+					article_img_url:
+						"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+					comment_count: 2,
+				});
+				expect(articles.body.articles).toBeSortedBy("created_at", {
```

Comment:
This is a different behaviour and so would be suitable to extract into its own test